src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -189,6 +192,7 @@ def __init__(
         timestep_type: str = "discrete",  # can be "discrete" or "continuous"
         steps_offset: int = 0,
         rescale_betas_zero_snr: bool = False,
+        final_sigmas_type: str = "zero",  # can be "zero" or "sigma_min"
```

Comment:
Type could be changed to `Literal["zero", "sigma_min"]` to be more precise.

Reply:
good idea! I would maybe ask the community to help us do this for the whole code base